euphoria-inmem/src/test/java/cz/seznam/euphoria/inmem/InMemExecutorTest.java

```diff
@@ -337,6 +337,11 @@ public boolean equals(Object o) {
     public int hashCode() {
       return size;
     }
+
+    @Override
+    public int compareTo(SizedCountWindow o) {
+      return 0;
+    }
```

Comment:
i think this is a nice example showing how _not_ to implement `compareTo` for a window; please make it consistent with the `equals` impl at least for the sake of demo purposes.